framework/tst/dslabs/framework/testing/newviz/SingleNodePanel.java

```diff
@@ -137,6 +145,11 @@ void updateState(EventTreeState s, boolean viewDeliveredMessages) {
             nodeState.clearDiffObject();
         }
 
+        boolean searchDone = settings != null &&
```

Comment:
This is also a product of my laziness, but there's more logic now. This code shouldn't be repeated. The constructor should call `updateState`.

Reply:
Fixed in https://github.com/emichael/dslabs/pull/29/commits/472418acfffa20d1d968e3e5a293784bd37e2bb0.